docs/releases/5.0.txt

```diff
@@ -279,6 +279,8 @@ Models
   :ref:`Choices classes <field-choices-enum-types>` directly instead of
   requiring expansion with the ``choices`` attribute.
 
+* Added ``django.models.fields.GeneratedField``.
```

Comment:
1. link to the appropriate docs
2. summarize the feature

Without these, the release note is not going to surface your hard work and make it easy for others to discover :)

```suggestion
* The new :class:`~django.db.models.fields.GeneratedField` allows creation of
  database generated columns.
```

Reply:
LGTM